tests/converter/test_biosamples_converter.py

```diff
@@ -0,0 +1,124 @@
+import json
+import unittest
+from os.path import dirname
+
+from biosamples_v4.models import Sample, Attribute
+from typing import List
+
+from converter.biosamples import BioSamplesConverter
+
+
+class BioSamplesConverterTests(unittest.TestCase):
+
+    def setUp(self) -> None:
+        with open(dirname(__file__) + '/../resources/biomaterials.json') as file:
+            self.biomaterials = json.load(file)
+
+        self.biosamples_converter = BioSamplesConverter()
+        self.domain = 'self.test_domain'
+
+    def test_given_ingest_biomaterial_when_release_date_defined_in_project_converts_correct_biosample(self):
```

Comment:
These tests look very similar.
Have you considered using a [test generator](https://nose.readthedocs.io/en/latest/writing_tests.html#test-generators)